app/default/routes.py

```diff
@@ -1,46 +1,104 @@
+from app.default.data import determine_round_status
+from app.default.data import get_default_round_for_fund
 from app.default.data import get_fund_data
+from app.default.data import get_fund_data_by_short_name
 from app.default.data import get_round_data
+from app.default.data import get_round_data_by_short_names
 from config import Config
+from flask import abort
 from flask import Blueprint
 from flask import current_app
+from flask import redirect
 from flask import render_template
 from fsd_utils.simple_utils.date_utils import (
     current_datetime_after_given_iso_string,
 )
 
-
 default_bp = Blueprint("routes", __name__, template_folder="templates")
 
 
+# TODO Do not change this until after COF R2 W3 closes as this
+# provides the current landing page!
 @default_bp.route("/")
```

Comment:
Can we make this a `302` redirect to the COF R2 W3 specific page? This way we could start using that without breaking any existing links?

Reply:
Oh yeah, good shout 👍